gradlew

```diff
@@ -92,6 +92,9 @@ fi
 # If the file still doesn't exist, let's try `wget` and cross our fingers
 if [ ! -e $APP_HOME/gradle/wrapper/gradle-wrapper.jar -a "$(command -v wget)" ]; then
     wget -O $APP_HOME/gradle/wrapper/gradle-wrapper.jar https://raw.githubusercontent.com/gradle/gradle/v8.10.0/gradle/wrapper/gradle-wrapper.jar
```

Comment:
In GitHub Action environment, DNS setup could fail like the following. So, we need to try to fallback to IP in the worst case.
```
#14 [builder 4/4] RUN ./gradlew clean build -x check
#14 5.254 wget: bad address 'raw.githubusercontent.com'
```